start.sh

```diff
@@ -9,7 +9,7 @@ if [[ -z "${MESOS_SLAVE_PID}" ]]; then
     source ~/.chs_env/global_env
     source ~/.chs_env/lfp-pay-api/env
 
-    PORT="${LFP_PAY_API_PORT}"
+    PORT="${PPS_PAY_API_PORT}"
```

Comment:
I'm not sure where this env variable comes from - assume it is in the reference source file above.
Possibly `global_env`?

I can only see something previously in chs-configs repo - but that appears under vagrant-chs-development/global_env.

Reply:
I am going to revert for now, and it can be updated when we are renaming the env stuff above.